CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Given a version number `MAJOR.MINOR.PATCH`, we increment the:
 
 ## Unreleased
 
+## 0.5.0
```

Comment:
This part you can leave out, only for non-RCs.

Reply:
👍🏾